src/modules/75-cf/components/CreateEnvironmentDialog/__tests__/EnvironmentDialog.test.tsx

```diff
@@ -74,6 +100,57 @@ describe('EnvironmentDialog', () => {
     })
   })
 
+  test('it should autofill the environment name when created from input', async () => {
+    const onCreateEnv = jest.fn()
+    renderComponent({ onCreate: onCreateEnv, createEnvFromInput: true, createEnvName: 'my environment name' })
+
+    userEvent.click(screen.getByRole('button', { name: 'createSecretYAML.create' }))
+
+    await waitFor(() => {
+      expect(createEnvironment).toBeCalled()
+      expect(screen.queryByText('cf.create.env.error')).not.toBeInTheDocument()
+    })
+  })
+
+  test('it should sanitise the Id based on the env name text provided', async () => {
+    const onCreateEnv = jest.fn()
+    renderComponent({ onCreate: onCreateEnv, createEnvFromInput: true, createEnvName: 'my environment name' })
```

Comment:
Looks like the same for this one, you can remove the `onCreate` override
```suggestion
    renderComponent({ createEnvFromInput: true, createEnvName: 'my environment name' })
```